ROADMAP.md

```diff
@@ -4,4 +4,138 @@ Roadmap
+types, and established the in-toto attestation maintainers, a group of industry
+stakeholders who review and provide feedback about new attestation types. ITE-7
+added support for creating and verifying signatures using X.509 certificates.
+
+### ITE-6, ITE-9, and Policies for in-toto Attestations
+
+This year, we are working towards having both of these ITEs accepted. ITE-6 is
+already used in a number of applications, most notably by the SLSA
+specification. We believe it's time to accept ITE-6 and continue to develop the
+attestation framework. Similarly, ITE-9 is still a draft but the process
+described in it is already being applied to several new attestation type
+proposals. The attestation maintainers have reviewed proposals for runtime
+traces and the Software Supply Chain Attribute Integrity (SCAI) specification.
+
+Besides these existing ITEs, a key aim for this year is to introduce a new ITE
+that proposes modifications to in-toto layouts. As it stands, in-toto layouts
+cannot be used to verify ITE-6 attestations. With ITE-6 close to being accepted,
+the focus is now on closing this gap so that in-toto attestations can be
+verified using our verification workflows. Alongside defining new layout
+schemas, we hope to revive the work on ITE-8 to introduce a library of in-toto
+policies.
+
+### Hierarchy for Attestation Types and the Predicate Dictionary
+
+In the last year, we have discussed at several times if we want to establish a
+hierarchy for in-toto attestation types. As more attestations are introduced for
+various contexts, some may be derived or may extend others. As such, one of the
+aims for this year is to formally study how to handle these scenarios.
+
+Another closely related discussion was about the creation of a "predicate
+dictionary". The idea here was to identify how different attestation types
+overlap, and when they can be used interchangeably. For example, a SLSA
+Provenance attestation contains every information used by in-toto's verification
+workflow from in-toto link metadata. We believe that building this out will go a
+long way to improving in-toto's usability and a sense of the minimum set of
+attestations necessary to secure software supply chains as per current
+standards.
+
+### ITE-7
+
+As noted before, this ITE introduced the use of X.509 certificates in in-toto.
+This ITE has been implemented in in-toto-golang and other implementations such
+as TestifySec's Witness. After another round of reviews at KubeCon late last
+year, this ITE is also nearing acceptance.
+
+## in-toto Attestations
+
+Apart from the task of reviewing and providing feedback for the development of
+new attestation types, the attestation maintainers also plan to work on updating
+and maintaining the in-toto attestation specification itself. The plan is to
+focus on the backlog of discussions and issues that have been ongoing in the
+last year or so. Further, the maintainers plan to improve the list of reviewed
+predicates.
```

Comment:
I think one important aspect that we might want to explicitly call out (since we've already been discussing this a bit at the maintainers meetings), is generic or language-independent tooling that the different in-toto implementations can use to generate and verify ITE-6/ITE-9 data. This tooling I think will really help make adoption of ITE-6/ITE-9 much more smoother and streamlined.